code/core/src/common/utils/paths.ts

```diff
@@ -17,10 +19,7 @@ export const getProjectRoot = () => {
   }
 
   try {
-    const found =
-      findUpSync('.git', { type: 'directory' }) ||
-      findUpSync('.svn', { type: 'directory' }) ||
-      findUpSync('.hg', { type: 'directory' });
+    const found = find.up('.git') || find.up('.svn') || find.up('.hg');
```

Comment:
to be strictly equal to the behaviour before, you probably want to use `find.dir` rather than `find.up`

Reply:
Yeah, it didn't exist when I implemented this :p